test/modules/aws-rds-integration.ts

```diff
@@ -45,7 +45,7 @@ describe('RDS Integration Testing', () => {
         VALUES ('${prefix}test', 20, 'db.t3.micro', 'test', 'testpass', '${availabilityZone}', 'postgres:13.4', 0);
       INSERT INTO rds_security_groups (rds_id, security_group_id) SELECT
         (SELECT id FROM rds WHERE db_instance_identifier='${prefix}test'),
-        (SELECT id FROM security_group WHERE group_name='default');
+        (SELECT id FROM security_group WHERE group_name='default' LIMIT 1);
```

Comment:
This isn't really the correct logic here, but it works. What is going on is a few test accounts accidentally have two or more VPCs defined that weren't cleaned up, and this code isn't also joining the security group on the default VPC.

Choosing a random VPC by selecting a random default security group is not something anyone would actually want to do outside of our own test suite, but copy-pasting happens, so I'd prefer to set a better example. :)

Reply:
oh yes, I will remove this, it also does not belong to this PR. It was only to avoid a test failure.